conda/plugins/subcommands/doctor/health_checks.py

```diff
@@ -163,8 +163,19 @@ def env_txt_check(prefix: str, verbose: bool) -> None:
     print(f"Environment listed in environments.txt file: {present}\n")
 
 
+def requests_ca_bundle_check(prefix: str, verbose: bool) -> None:
+    import os
```

Comment:
stdlib imports should be left at the top of the file

Reply:
@kenodegard , then we don't need the following two lines, right?
https://github.com/conda/conda/blob/main/conda/plugins/subcommands/doctor/health_checks.py#L19-L20